server/root_ca.go

```diff
@@ -9,8 +9,8 @@ import (
 	"github.com/pkg/errors"
 )
 
-// ServeHTTPRootCA starts a HTTP server running on `address`
-// serving the root CA from "/ca". It rejects all other requests.
+// ServeHTTPRootCA starts a HTTP server running on `address` using the .pem file at
+// rootCA, serving the root CA from "/ca". It rejects all other requests.
```

Comment:
The server is not "using" any `.pem` files, it's an HTTP server. The main server however does ;)

Reply:
switch "using" to "serving"